### PR TITLE
Rename `get-started-experiments` to `example-dvc-experiments`

### LINKED ISSUE #79: rename `get-started-experiments` to `dvc-example-experiments`.
We have a new naming convention for the example projects and the latter name seems more descriptive.

### PR DESCRIPTION
**Pull request by iesahin**
This renames the project name to `example-dvc-experiments`. It also changes some internal names and adds codespaces support to the produced repository. 

Related #74
Closes #79

### CODE REVIEW

File: example-dvc-experiments/code/.devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "example-dvc-experiments",
+    "build": {
+        "dockerfile": "Dockerfile"
+    }
+}
```
Comment (shcheklein) [inline example-dvc-experiments/code/.devcontainer/devcontainer.json:6]: newline

Comment (iesahin) [reply example-dvc-experiments/code/.devcontainer/devcontainer.json:6]: https://www.loom.com/share/d78aac3bf8c1489eac79e1da7e0550d5 :)

Comment (shcheklein) [reply example-dvc-experiments/code/.devcontainer/devcontainer.json:6]: loom! new way to communicate, I like it (for real) :)

I meant something like this https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file ... it's not usually about language specifically, it's about in general having end of lines at the end for UNIX tools to be have properly - just a best practice I guess, that's why GH highlights these missing newlines

not a big deal, clearly :)

Comment (iesahin) [reply example-dvc-experiments/code/.devcontainer/devcontainer.json:6]: I'm also in the _EOL at the EOF_ camp and I believe it's standard but these days standards are determined by the editors I think.

**Review by casperdcl — COMMENTED** [review]
lgtm

**Pull request merged**